conf/modules.config

```diff
@@ -57,11 +57,32 @@ process {
         ]
     }
 
+    withName: ENSEMBLVEP_VEP {
+        container = 'ensemblorg/ensembl-vep:release_111.0' //sticking to v111 for now, but we should update this
```

Comment:
This version should match the cache_version  or at least this one can't be inferior to the cache version, maybe we need to define the  container version in the module ..

Reply:
We were using version 111 before. If I understand this comment properly, our current vep reference data in cqdg match version 112 and not 111. I will try to figure out which vep version is the reference data. There might be a metadata file / checksum that I can use.

Reply:
Pour l'instant, je vois que le nom du folder qui contient les données de vep dans le minio de CQDG est bien nommé selon la version 111:
cqdg-prod-file-import/nextflow/reference/annotation/vep/homo_sapiens/111_GRCh38/

Je suis en train de télécharger le fichier de vep pour comparer. La vitesse de téléchargement est très lente. Ça va probablement prendre toute la nuit.

Reply:
J'ai pu voir ce matin que la taille du vep cache qu'on a sur JUNO est plus grande (environ 26G) que celle de la version 111 (environ 23G). 

Je crois qu'il s'agit probablement de la version 112.  Je vais corriger la documentation pour que ce soit clair et je vais ouvrir un ticket séparé pour fixer ça.